README.md

```diff
@@ -1,27 +1,86 @@
 [![Build Status](https://travis-ci.org/AdoptOpenJDK/openjdk-infrastructure.svg?branch=master)](https://travis-ci.org/AdoptOpenJDK/openjdk-infrastructure)
+
+
+# Mission Statement
+
+To provide infrastructure for the AdoptOpenJDK farm that is:
+
+* **Secure** - Infrastructure is private by default and access is granted in a 
+time and access control limited manner.
+* **Consistent** - Infrastructure is consistent in order to produce consistent 
+AdoptOpenJDK binaries.
+* **Repeatable** - Infrastructure can be reproduced by our _infrastrucure as code_. 
+We embrace the Chaos Monkey.
+* **Auditable** - What each host/platform is made up of is publicly accessible 
+_infrastructure as code_.
+
+The end result should be **immutable** hosts, which can be destroyed and reproduced from Ansible playbooks. See 
+our [Contribution Guidelines](https://www.github.com/adoptopenjdk/openjdk-infrastructure/Contributing.md) 
+on how we implement these goals.
+
+## Can we Chaos Monkey it?
+
+See our current [Chaos Monkey Status](CHAOS_MONKEY.md).
+
+# Related Repos
+
+* [email](https://www.github.com/adoptopenjdk/email/) - A repo containing configuration for our email aliases etc.
+* [secrets](https://www.github.com/adoptopenjdk/secrets/) - A private repo containing encrypted secrets.
+* [openjdk-jenkins-helper](https://www.github.com/adoptopenjdk/openjdk-jenkins-helper/) - A repo containing helper scripts for out Jenkins CI.
+
+## Important Documentation
+
+* [hosts](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/inventory.yml) - Our inventory, [visualised](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/docs/adoptopenjdk.pdf).
+* [Ansible at AdoptOpenJDK](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/README.md) - Our hosts are built using Ansible Playbooks. 
+
+# Contributing
+
+Please visit our `#infrastructure` [Slack Channel](https://www.adoptopenjdk.net/slack.html) and say hello. 
+Please read our [Contribution Guidelines](https://www.github.com/adoptopenjdk/openjdk-infrastructure/Contributing.md) before 
+submitting Pull Requests.
+
 # Members
 
-#### [@admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure)
+We list members and their organisation affiliation for maximum transparency. Want to add 
+a new member? Please follow our [Onboarding Process](ONBOARDING.md). 
 
-- [@gdams](https://github.com/gdams) - George Adams (IBM)
-- [@johnoliver](https://github.com/johnoliver) - John Oliver (jClarity / LJC)
-- [@mwornast](https://github.com/mwornast) - Marcus Wornast (IBM)
+## [@admin_infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/admin_infrastructure)
```

Comment:
Maybe I've missed it but what's the `*` against someone's name intended to indicate?

Reply:
Access to the secrets repo.  I need to document that.